packages/credential-provider-web-identity/package.json

```diff
@@ -15,7 +15,6 @@
     "test": "jest"
   },
   "browser": {
-    "./dist-cjs/fromTokenFile": false,
```

Comment:
Make script more restrictive to check if the value is a string, and ignore if it's boolean?

This is needed as for webpack support https://github.com/aws/aws-sdk-js-v3/pull/3005

Reply:
we don't provide dist-cjs corrections in other packages for the browser environment

Reply:
added condition to skip dropping boolean directives in browser with CJS to avoid unforeseen consequences